### PR TITLE
Add ChirpStack helm chart

### PR DESCRIPTION
**Pull request by Kuruyia**
## What does this PR do ?
This adds the helm chart for deploying ChirpStack and its REST API bridge.

ChirpStack dependencies (Postgres, Redis & an MQTT broker) must be deployed separately, and the hosts of those services must be set accordingly when installing this chart.

### CODE REVIEW

**Comment by Aschen** [comment]
I don't understand this sentence ` the hosts of those services must be set accordingly when installing this chart.`

Does it means that Chirpstack can be deployed on each client?

**Comment by Kuruyia** [comment]
In our case, ChirpStack will not be deployed on each client but globally (although the helm chart has been written so it could be configured to be installed anywhere).

This sentence meant that, when installing the helm chart, you have to manually set the hostnames where ChirpStack can connect to the Postgres, the Redis and the MQTT broker.

File: charts/chirpstack/templates/chirpstack/cm.yaml [hunk 1]
```diff
@@ -0,0 +1,64 @@
+apiVersion: v1
```
Comment (rolljee) [inline charts/chirpstack/templates/chirpstack/cm.yaml:1]: You can give this file the proper fullname

File: charts/chirpstack/templates/chirpstack/cm.yaml [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+kind: ConfigMap
+metadata:
+  name: {{ template "chirpstack.core" . }}
+  labels:
+    {{- include "chirpstack.labels" . | nindent 4 }}
+data:
+  chirpstack.toml: |
+    [logging]
```
Comment (rolljee) [inline charts/chirpstack/templates/chirpstack/cm.yaml:9]: Could you add a link to the docs of this toml file in a comment

File: charts/chirpstack/templates/chirpstack/cm.yaml [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+      secret="{{ .Values.chirpstack.apiSecret }}"
+
+
+    {{- if .Values.chirpstack.metrics.enabled }}
+    [monitoring]
+      bind="0.0.0.0:{{ .Values.chirpstack.metrics.port }}"
+    {{ end }}
+
+  eu868.toml: |
+    [[regions]]
```
Comment (rolljee) [inline charts/chirpstack/templates/chirpstack/cm.yaml:52]: Same here, add a link to this docs

File: charts/chirpstack/values.yaml [hunk 4]
```diff
@@ -0,0 +1,180 @@
...
+    sslMode: disable
+
+  redis:
+    servers:
+      - redis://localhost/
+
+    tlsEnabled: false
+    cluster: false
+
+  regions:
```
Comment (rolljee) [inline charts/chirpstack/values.yaml:43]: This key name seems odd, region is rarely used for host configuration

**Review by rolljee — CHANGES_REQUESTED** [review]

**Review by rolljee — APPROVED** [review]

**Pull request merged**